test/analysis/test_utils.py

```diff
@@ -2,14 +2,39 @@
 
 import numpy as np
 
-from snorkel.analysis.utils import filter_labels, preds_to_probs, probs_to_preds
+from snorkel.analysis.utils import (
+    arraylike_to_numpy,
+    convert_labels,
+    filter_labels,
+    preds_to_probs,
+    probs_to_preds,
+)
 
 PROBS = np.array([[0.1, 0.9], [0.7, 0.3]])
 PREDS = np.array([2, 1])
 PREDS_ROUND = np.array([[0, 1], [1, 0]])
 
 
 class MetricsTest(unittest.TestCase):
+    def test_bad_input(self):
+        not_array = "112200"
+        Y_nparray = np.ones((3,))
+        self.assertRaises(ValueError, arraylike_to_numpy, not_array)
+        self.assertRaises(
+            ValueError,
+            convert_labels,
+            Y=not_array,
+            source="categorical",
+            target="plusminus",
+        )
+        self.assertIsNone(
+            convert_labels(Y=None, source="categorical", target="plusminus")
+        )
+        np.testing.assert_array_equal(
```

Comment:
It looks like the last test is a working test, not a bad input.

Reply:
changing name to`test_input`